<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuddiesClass</class>
 <widget class="QMainWindow" name="BuddiesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset resource="Buddies.qrc">
         <normaloff>:/images/Usergroup.png</normaloff>:/images/Usergroup.png</iconset>
       </attribute>
       <attribute name="title">
        <string>(B)*</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="Buddies.qrc">:/images/Logo.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Buddies Network</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="labelLogin">
             <property name="text">
              <string>Login:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditLogin</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditLogin"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelPasswd">
             <property name="text">
              <string>Password:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditPasswd</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditPasswd">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="StatusLabel">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="Buddies.qrc">:/images/Error.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCommandLinkButton" name="commandLinkButton">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>169</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="icon">
        <iconset resource="Buddies.qrc">
         <normaloff>:/images/Man.png</normaloff>:/images/Man.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Online</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget">
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="icon">
        <iconset resource="Buddies.qrc">
         <normaloff>:/images/Options.png</normaloff>:/images/Options.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Baloon Messages</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="labelDuration">
             <property name="text">
              <string>Choose message duration:</string>
             </property>
             <property name="buddy">
              <cstring>durationSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="durationSpinBox">
             <property name="value">
              <number>15</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QLabel" name="labelIP">
               <property name="text">
                <string>IP Address:</string>
               </property>
               <property name="buddy">
                <cstring>lineEditIP</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditIP">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelHostname">
               <property name="text">
                <string>Hostname:</string>
               </property>
               <property name="buddy">
                <cstring>lineEditHost</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditHost">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBoxDbStatus">
          <property name="title">
           <string>Database status</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QLabel" name="labelDbStatus">
             <property name="text">
              <string>Buddies Database:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelimageStatus">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Status:</string>
      </property>
      <property name="buddy">
       <cstring>iconComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="iconComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit_2"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Configure"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_Credits"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionChat"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="separator"/>
   <addaction name="actionBoard"/>
   <addaction name="separator"/>
   <addaction name="actionNotes"/>
  </widget>
  <action name="action_Credits">
   <property name="icon">
    <iconset resource="Buddies.qrc">
     <normaloff>:/images/Welcome.png</normaloff>:/images/Welcome.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Credits</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="Buddies.qrc">
     <normaloff>:/images/Exit.png</normaloff>:/images/Exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="icon">
    <iconset resource="Buddies.qrc">
     <normaloff>:/images/Chat.png</normaloff>:/images/Chat.png</iconset>
   </property>
   <property name="text">
    <string>chat</string>
   </property>
   <property name="toolTip">
    <string>Team Chat</string>
   </property>
  </action>
  <action name="action_Exit_2">
   <property name="icon">
    <iconset resource="Buddies.qrc">
     <normaloff>:/images/Disconnect.png</normaloff>:/images/Disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="Buddies.qrc">
     <normaloff>:/images/Options.png</normaloff>:/images/Options.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="action_Configure">
   <property name="icon">
    <iconset resource="Buddies.qrc">
     <normaloff>:/images/Options.png</normaloff>:/images/Options.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
  </action>
  <action name="actionBoard">
   <property name="icon">
    <iconset resource="Buddies.qrc">
     <normaloff>:/images/Bubbles.png</normaloff>:/images/Bubbles.png</iconset>
   </property>
   <property name="text">
    <string>Board</string>
   </property>
   <property name="toolTip">
    <string>Wall</string>
   </property>
  </action>
  <action name="actionNotes">
   <property name="icon">
    <iconset resource="Buddies.qrc">
     <normaloff>:/images/Pin.png</normaloff>:/images/Pin.png</iconset>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
   <property name="toolTip">
    <string>Notes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Buddies.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>commandLinkButton</sender>
   <signal>clicked()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>loginUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPasswd</sender>
   <signal>returnPressed()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>loginUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Exit</sender>
   <signal>activated()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>myexit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Exit_2</sender>
   <signal>activated()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>myexit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit</sender>
   <signal>activated()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>focusTabEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChat</sender>
   <signal>activated()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>startChat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Credits</sender>
   <signal>activated()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>helpAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Configure</sender>
   <signal>activated()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>focusTabEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>BuddiesClass</receiver>
   <slot>startChat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBoard</sender>
   <signal>activated()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>WriteOnWall()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNotes</sender>
   <signal>activated()</signal>
   <receiver>BuddiesClass</receiver>
   <slot>openNotes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginUser()</slot>
  <slot>myexit()</slot>
  <slot>focusTabEdit()</slot>
  <slot>structUser()</slot>
  <slot>startChat()</slot>
  <slot>helpAbout()</slot>
  <slot>WriteOnWall()</slot>
  <slot>Information()</slot>
  <slot>openNotes()</slot>
 </slots>
</ui>
